metricbeat/module/mysql/status/_meta/fields.yml

```diff
@@ -160,3 +160,183 @@
           type: long
           description: >
             The number of UPDATE queries since startup.
+
+    - name: queries
+      type: long
+      description: >
```

Comment:
Thanks for contributing! Could you add some descriptions for each new field please? That would be helpful!